deps/libgit2.mk

```diff
@@ -84,6 +84,11 @@ $(LIBGIT2_SRC_PATH)/libgit2-mbedtls-writer-fix.patch-applied: $(LIBGIT2_SRC_PATH
 		patch -p1 -f < $(SRCDIR)/patches/libgit2-mbedtls-writer-fix.patch
 	echo 1 > $@
 
+$(LIBGIT2_SRC_PATH)/libgit2-mbedtls-verify.patch-applied: $(LIBGIT2_SRC_PATH)/source-extracted
```

Comment:
it would be better to make the dependencies explicit, there would be a race condition if both patches get applied at the same time, one of the applications could get overwritten by the other